topics/introduction/metadata.yaml

```diff
@@ -248,6 +248,13 @@ material:
       - "Gain understanding on navigating and manipulating histories"
     time_estimation: "30 min"
     key_points:
+      - In Galaxy, data is stored in datasets and organized in histories.
+      - The right pane shows the current history. It displays the name of the history, the tags and annotations assigned to it, and the size of the data in it.
+      - The datasets are shown differently depending on their status: queued, running, ok, error, or paused.
```

Comment:
could you put quotes around this line? (since it contains a colon)

Reply:
Sure. Hope it looks better now.

Reply:
sorry, I meant quote the entire line (so like the objectives and questions above it) this is because since the colon has a special meaning in yaml, the parser will not understand it without the quotes.

Reply:
I see. I changed it.